Applications/_Plugins/Common/CMP_FileIO.h

```diff
@@ -18,21 +18,6 @@
 #ifndef _CMP_FILEIO_H_
 #define _CMP_FILEIO_H_
 
-#ifdef _WIN32
```

Comment:
It's super important to try and reduce the number of headers included in other header files. This can lead to compilation errors that feel impossible to track down.